properties/base/instance.py

```diff
@@ -101,8 +101,14 @@ def validate(self, instance, value):
             if isinstance(value, dict):
                 return self.instance_class(**value)
             return self.instance_class(value)
-        except (ValueError, KeyError, TypeError):
-            self.error(instance, value)
+        except GENERIC_ERRORS as err:
+            if hasattr(err, 'error_tuples'):
```

Comment:
can't we do this through checking the subclass rather than adding a attribution dynamically?

Reply:
That makes sense, `isinstance(err, ValidationError)` is better than duck typing.